stories/shadcn/multi-select.stories.tsx

```diff
@@ -0,0 +1,62 @@
+"use client"
+import type { Meta, StoryObj } from "@storybook/react";
+import React, { ComponentProps } from "react";
+
+import {
+  MultiSelect,
+  MultiSelectContent,
+  MultiSelectEmpty,
+  MultiSelectGroup,
+  MultiSelectItem,
+  MultiSelectList,
+  MultiSelectSearch,
+  MultiSelectTrigger,
+  MultiSelectValue,
+} from "src"
+
+
+
+const Demo = (args: ComponentProps<typeof MultiSelect>) => {
+  const [value, setValue] = React.useState(args.value);
+//   args.
```

Comment:
```suggestion
```

Reply:
Feel free to edit and push to this branch yourself. Let me know when you're ready for a review.

https://github.com/shadcn-ui/ui/pull/2773
https://ui-git-fork-yeliex-feat-multi-select-shadcn-pro.vercel.app/docs/components/multi-select